test/check-application

```diff
@@ -1018,12 +1018,13 @@ class TestApplication(testlib.MachineCase):
             self.execute(True, f"podman rmi -f {IMG_BUSYBOX} localhost:5000/my-busybox localhost:6000/my-busybox")
             self.execute(False, f"podman rmi -f {IMG_BUSYBOX}")
 
-        class DownloadImageDialog:
-            def __init__(self, imageName, imageTag=None, user="system"):
+        class DownloadImageDialog():
+            def __init__(self, test_obj, imageName, imageTag=None, user="system"):
                 self.imageName = imageName
                 self.imageTag = imageTag
                 self.user = user
                 self.imageSha = ""
+                self.assertTrue = test_obj.assertTrue
```

Comment:
This is really an anti-pattern but I'll allow it. This makes it look like it's a unittest class while it's not.